content/blog/2023-11-13-announcing-localstack-30-general-availability/index.md

```diff
@@ -0,0 +1,186 @@
+
+Check out our [Step Functions documentation](https://docs.localstack.cloud/user-guide/aws/stepfunctions/) and [Discuss post](https://discuss.localstack.cloud/t/new-stepfunctions-implementation-in-localstack-3-0/593) for more information.
+
+### New ElastiCache provider
+
+We have introduced a **new ElastiCache provider** with better parity with AWS and resolves issues around the Redis cluster management in the previous provider. The new provider implements a dedicated Redis control plane to manage Redis instances in non-cluster mode and cluster mode. The new provider will be extended in the future to test automatic failover, snapshotting, scaling replication groups, and more features.
+
+These latest enhancements mark a significant step forward in our commitment to delivering a more robust and fine-tuned ElastiCache provider. The old ElastiCache provider is temporarily available in LocalStack v3 using `PROVIDER_OVERRIDE_ELASTICACHE=legacy` but we highly recommend migrating as soon as possible since we will drop support for this in LocalStack v4.
+
+### New features for Chaos Engineering
+
+Chaos engineering is a practice focused on improving system resilience by intentionally introducing disruptions. To support this, we're introducing a new **Chaos Engineering** dashboard in the LocalStack Web Application. This feature allows users to conduct fault injection experiments within their application stack. The dashboard offers various Fault Injection Simulator (FIS) experiment options, such as:
+
+- **500 Internal Error**: This experiment randomly terminates incoming requests and returns an `Internal error` with a response code of 500.
+- **Service Unavailable**: Similar to the previous one, this experiment randomly terminates incoming requests but responds with a `Service Unavailable` message and a 503 response code.
+- **AWS Region Unavailable**: This simulates region outages and failovers by randomly terminating incoming requests to mimic regional service disruptions.
+- **Latency**: This introduces a specified amount of latency to each API call, useful for simulating slow network conditions or network performance issues.
+
+{{< img-simple src="localstack-chaos-engineering-dashboard.png" width=300 alt="Image of LocalStack Chaos Engineering dashboard">}}
+
+Additionally, we have introduced user guides for various scenarios, including simulating unexpected outages using the [LocalStack Outage Extension](), implementing [Route53 Failover with FIS](), and configuring error probabilities in [Kinesis]() & [DynamoDB]().
+
+These resources are designed to help users effectively respond to such scenarios, facilitating the development of thorough disaster recovery plans. By using these tools, teams can prepare to sustain stability and efficiency even in challenging situations. Check out our [documentation]() for more information.
```

Comment:
link missing

Reply:
They are not merged yet hence not available!